default-tiles/status-bar/src/main.rs

```diff
@@ -2,7 +2,7 @@ mod first_line;
 mod second_line;
 
 use std::fmt::{Display, Error, Formatter};
-use zellij_tile::*;
+use zellij_tile::prelude::*;
```

Comment:
Personally I'm not a fan of wildcard imports. It makes it harder to figure out "where this thing is coming from", but that's just my opinion. :)

Reply:
I agree as a general rule and am using specific imports within Zellij, but I figured that it would be nice to have a single import for plugins. I changed the wild-card import to a `prelude` to follow the "convention" for wildcard imports (as opposed to just having the earlier `zellij_tile::*` wildcard).

I'll probably keep this as is for now, but when you are writing things, feel free to import from specific modules! :)